boards/nucleo_h563zi.json

```diff
@@ -0,0 +1,48 @@
+{
+  "build": {
+    "core": "stm32",
+    "cpu": "cortex-m33",
+    "extra_flags": "-DSTM32H5 -DSTM32H5xx -DSTM32H563xx",
+    "f_cpu": "250000000L",
+    "mcu": "stm32h563zit6u",
+    "product_line": "STM32H563xx",
+    "variant": "STM32H5xx/H563Z(G-I)T_H573ZIT"
+  },
+  "connectivity": [
+    "can",
+    "ethernet"
+  ],
+  "debug": {
+    "default_tools": [
+      "stlink"
+    ],
+    "jlink_device": "STM32H723IE",
+    "onboard_tools": [
+      "stlink"
+    ],
+    "openocd_target": "stm32h7x",
```

Comment:
The current OpenOCD package doesn't support the `stm32h5` family, so it should be removed completely.